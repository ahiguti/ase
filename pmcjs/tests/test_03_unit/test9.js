
try {
  Perl.Functions[""].abc();
} catch (e) {
  print("type=" + e.etype);
  print("error=" + e.error);
  print("func=" + e.func);
  print("description=" + e.description);
  print("string=" + e);
}

