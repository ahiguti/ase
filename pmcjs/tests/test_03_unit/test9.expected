type=RuntimeException
error=PerlException
func=pmc_funccall_internal
description=Undefined subroutine &main::abc called. 
string=(
 etype: RuntimeException,
 error: PerlException,
 func: pmc_funccall_internal,
 description: Undefined subroutine &main::abc called. ,
 at: 
  func=[] file=test9.js line=3
)
